src/core/agent/services/delegationService.ts

```diff
@@ -70,7 +70,7 @@ class DelegationService extends AgentService {
       .get(metadata.signifyName);
     const operation = await this.props.signifyClient
       .keyStates()
-      .query(identifier.state.di, "1");
+      .query(identifier.state.di as string, "1");
```

Comment:
it would be better to make sure `di` is not undefined - we don't do delegation right now but we might update from this code so safer to update it now...

Reply:
@iFergal Should we throw an error if this `di` is `undefined`?